packages/ui/src/components/molecules/LeanBox.tsx

```diff
@@ -119,8 +148,14 @@ export const LeanBox: FC<LeanBoxProps> = ({
         )}
         {tag && <Tag>{tag}</Tag>}
         <RightContent>
+          {throbber && <Throbber size={15} strokeWidth={2} />}
```

Comment:
This should be taken care of by `rightImage`, no need for a separate throbber prop. Check how it's used in `AddAccount`

Reply:
done